docs/source/distributed.md

```diff
@@ -81,8 +83,9 @@ In the past, we were often asked: "which backend should I use?".
 
 - Rule of thumb
 
-  - Use the NCCL backend for distributed **GPU** training
-  - Use the Gloo backend for distributed **CPU** training.
+  - Use the NCCL backend for distributed training with CUDA-enabled **GPU**.
+  - Use the XCCL backend for distributed training with XPU-enabled **GPU**.
```

Comment:
Just "XPU" instead of "XPU-enabled GPU" to make it straightforward?

Reply:
Replied separately here https://github.com/pytorch/pytorch/pull/155834#issuecomment-3047398241